app/javascript/readingList/readingList.jsx

```diff
@@ -187,17 +194,17 @@ export class ReadingList extends Component {
       ''
     );
     return (
-      <section>
-        <header className="crayons-layout flex justify-between items-center pb-0">
+      <main>
```

Comment:
suggestion (not-blocking) - if we add the id `main-content` here, the skip link would work. Could also be picked up in a separate PR though as it's not really related to this issue

```suggestion
      <main id="main-content">
```